test/flows/business_coronavirus_support_finder_flow_test.rb

```diff
@@ -0,0 +1,260 @@
+  end
+
+  context "question: business_size?" do
+    setup do
+      testing_node :business_size?
+      add_responses business_based?: "england"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      %w[0_to_249 over_249].each do |response|
+        should "have a next node of paye_scheme? for an #{response} response" do
+          assert_next_node :paye_scheme?, for_response: response
+        end
+      end
+    end
+  end
+
+  context "question: paye_scheme?" do
+    setup do
+      testing_node :paye_scheme?
+      add_responses business_based?: "england",
+                    business_size?: "0_to_249"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      %w[yes no].each do |response|
+        should "have a next node of self_employed? for a #{response} response" do
+          assert_next_node :self_employed?, for_response: response
+        end
+      end
+    end
+  end
+
+  context "question: self_employed?" do
+    setup do
+      testing_node :self_employed?
+      add_responses business_based?: "england",
+                    business_size?: "0_to_249",
+                    paye_scheme?: "yes"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      %w[yes no].each do |response|
+        should "have a next node of non_domestic_property? for a #{response} response" do
+          assert_next_node :non_domestic_property?, for_response: response
+        end
+      end
+    end
+  end
+
+  context "question: non_domestic_property?" do
+    setup do
+      testing_node :non_domestic_property?
+      add_responses business_based?: "england",
+                    business_size?: "0_to_249",
+                    paye_scheme?: "yes",
+                    self_employed?: "yes"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      %w[yes no not_sure].each do |response|
+        should "have a next node of sectors? for a #{response} response" do
+          assert_next_node :sectors?, for_response: response
+        end
+      end
+    end
+  end
+
+  context "question: sectors?" do
+    setup do
+      testing_node :sectors?
+      add_responses business_based?: "england",
+                    business_size?: "0_to_249",
+                    paye_scheme?: "yes",
+                    self_employed?: "yes",
+                    non_domestic_property?: "yes"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      %w[nurseries retail_hospitality_or_leisure nightclubs_or_adult_entertainment personal_care none].each do |response|
+        should "have a next node of closed_by_restrictions? for a #{response} response" do
+          assert_next_node :closed_by_restrictions?, for_response: [response]
```

Comment:
```suggestion
          assert_next_node :closed_by_restrictions?, for_response: response
```

Reply:
`sectors?` is a checkbox question. If more than one answer is selected, they are sent as an array, so whilst a string response for a single answer does work, I thought it would be more confusing for one test for `sectors?` to pass in a string and the other one below to pass an array.

Reply:
@leenagupte yeah that makes sense!